ocean/nemo/nemolite2d/manual_versions/psykal_omp/Makefile

```diff
@@ -0,0 +1,83 @@
+# Makefile for manual OpenMP version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
+# environment variables. e.g. to use gfortran:
+# 
+# export F90=gfortran
+# export F90FLAGS="-O3"
+# export OMPFLAGS=" -fopenmp"
+# export AR=ar
+
+# Location of the dl_timer and infrastucture code
+SHARED_DIR = ../../../../../shared
+
+TIMER_DIR = ${SHARED_DIR}/dl_timer
+TIMER_INC = ${TIMER_DIR}/src
+TIMER_LIB = ${TIMER_DIR}/dl_timer_lib.a
+INF_DIR = ${SHARED_DIR}/dl_esm_inf/finite_difference
+INF_INC = ${INF_DIR}/src
+INF_LIB = ${INF_DIR}/src/dl_esm_inf_fd.a
+
+# The targets that this Makefile supports
+EXECS = nemolite2d
+
+# The modules that are common to both targets
+MODULES = model_mod.o boundary_conditions_mod.o \
+          continuity_mod.o initialisation_mod.o \
+          momentum_mod.o physical_params_mod.o \
+          time_update_mod.o gocean2d_io_mod.o \
+          time_step_mod.o
+
+GENERATED_MODULES = psy.o
+
+# API lib is an archive that must come at the end of the list of objects
+# passed to the linker
+COMMON_MODULES = $(MODULES) ${INF_LIB}
+
+timer_lib:
+	make -C ${TIMER_DIR} sm_lib
+
+inf_lib:
+	make -C ${INF_DIR}
+
+all: $(EXECS)
```

Comment:
this should be above timer_lib, otherwise it won't be compiled with make.

Reply:
Doh.